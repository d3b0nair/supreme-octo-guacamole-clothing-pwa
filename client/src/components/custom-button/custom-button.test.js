import React from "react";
import { render, cleanup } from "@testing-library/react";
import CustomButton from "./custom-button.component";

const mockProps = (otherProps) => ({ ...otherProps, children: "" });
const component = (mockProps) => <CustomButton {...mockProps} />;

afterEach(cleanup);

describe("CustomButton component", () => {
  it("should mount", () => {
    const { container } = render(component(mockProps()));
    expect(container).toMatchSnapshot();
  });
});
