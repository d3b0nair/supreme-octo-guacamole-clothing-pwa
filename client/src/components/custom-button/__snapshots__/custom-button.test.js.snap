// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton component should mount 1`] = `
<div>
  <button
    class="sc-bdvvtL fNaNEb"
  />
</div>
`;
